### PR TITLE
MWPW-136173 : Blog Sitemaps automation test script

### PR DESCRIPTION
**Pull request by skumar09**
[MWPW-136173 ](https://jira.corp.adobe.com/browse/MWPW-136173 ): Blog Sitemaps automation test script

PR includes 
- Automation test scripts for the following sitemap.xmls

  -   '/en/sitemap.xml',
  -  '/de/sitemap.xml',
  -  '/ko/sitemap.xml',
  -  '/es/sitemap.xml',
  -  '/fr/sitemap.xml',
  - '/it/sitemap.xml',
  - '/jp/sitemap.xml',
  - '/br/sitemap.xml',

### CODE REVIEW

File: tests/blog/sitemap.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        const response = await page.goto(`${baseURL}${path}`, { waitUntil: 'domcontentloaded' });
+        const statusCode = response.status();
+        expect(statusCode).toBe(200);
+        console.info(`HTTP Status Code: ${statusCode}`);
+      });
+
+      await test.step('step-2: verify hreflang attribute', async () => {
+        let lang = path.split('/')[1];
+        lang = (lang === 'jp' ? 'ja' : (lang === 'br' ? 'pt' : lang));
+        console.info(lang);
```
Comment (chadsunvice) [inline tests/blog/sitemap.test.js:21]: Is this line needed? Maybe make it a bit more verbose as to what the output means.

File: tests/blog/sitemap.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      await test.step('step-1: verify site map url response', async () => {
+        const response = await page.goto(`${baseURL}${path}`, { waitUntil: 'domcontentloaded' });
+        const statusCode = response.status();
+        expect(statusCode).toBe(200);
+        console.info(`HTTP Status Code: ${statusCode}`);
+      });
+
+      await test.step('step-2: verify hreflang attribute', async () => {
+        let lang = path.split('/')[1];
+        lang = (lang === 'jp' ? 'ja' : (lang === 'br' ? 'pt' : lang));
```
Comment (chadsunvice) [inline tests/blog/sitemap.test.js:20]: Maybe in the future we would need a file to have the LOCALE-to-LANGUAGE mappings imported, so we don't have to do these operations. Thoughts @nateekar @amauch-adobe?

**Review by chadsunvice — APPROVED** [review]
Looks good! Few mentions inline.

File: tests/blog/sitemap.test.js (hunk 2 above)
Comment (skumar09) [reply tests/blog/sitemap.test.js:20]: @chadsunvice : Yeah, I agree, that makes sense,

Comment (amauch-adobe) [reply tests/blog/sitemap.test.js:20]: I like the idea, let's do it.

File: envs/envs.js [hunk 3]
```diff
@@ -12,7 +12,6 @@ module.exports = {
   '@bacom_prod': 'https://business.adobe.com',
   '@bacom_stage': 'https://business.stage.adobe.com',
   '@blog_prod': 'https://blog.adobe.com',
-  '@blog_live': 'https://main--blog--adobe.hlx.live',
```
Comment (amauch-adobe) [inline envs/envs.js:15]: Are there any tests that use this before it gets removed/updated?

**Review by amauch-adobe — APPROVED** [review]
Approved, but see my comment on envs.js.

**Pull request merged**